sonar-project.properties

```diff
@@ -11,3 +11,4 @@ sonar.exclusions=tests/**
 sonar.python.coverage.reportPaths=coverage.xml
 sonar.python.xunit.reportPath=coverage.xml
 sonar.python.version=3
+sonar.coverage.exclusions=mpt_extension_sdk/**
```

Comment:
Please do not exclude coverage for all the SDK :-D I can agree to exclude it only for curtain peaces that you are moving from Adobe extension/working one for now as a temporary thing